doc/format.md

```diff
@@ -501,6 +502,33 @@ Example rule elements:
     bytes: 01 14 02 00 00 00 00 00 C0 00 00 00 00 00 00 46 = CLSID_ShellLink
     bytes: EE 14 02 00 00 00 00 00 C0 00 00 00 00 00 00 46 = IID_IShellLink
 
+### com
+COM features represent Component Object Model (COM) interfaces and classes used in the program's logic. They help identify interactions with COM objects, methods, properties, and interfaces. The parameter is the name of the COM.
+
+Example:
+
+```yaml
+- com/class: InternetExplorer #bytes: 01 DF 02 00 00 00 00 00 C0 00 00 00 00 00 00 46 = CLSID_InternetExplorer
+- com/interface: IWebBrowser2 #bytes: 61 16 0C D3 AF CD D0 11 8A 3E 00 C0 4F C9 E2 6E = IID_IWebBrowser2
+```
+
+Rule parsers will translate the above rule to its `bytes` and `string` representation by fetching the GUIDs from the COM database using the COM name.
+
+Translated Representation:
+
+```yaml
+- or:
+  - string : "0002DF01-0000-0000-C000-000000000046"
+    description: InternetExplorer as guid string
+  - bytes : 01 DF 02 00 00 00 00 00 C0 00 00 00 00 00 00 46 = InternetExplorer as bytes
```

Comment:
```suggestion
  - string : "0002DF01-0000-0000-C000-000000000046"
    description: InternetExplorer as GUID string
  - bytes : 01 DF 02 00 00 00 00 00 C0 00 00 00 00 00 00 46 = InternetExplorer as bytes
```

Reply:
if exported like this we could also use `CLSID_` and `IID_` prefixes

Reply:
Added COM prefixes